### ISSUE TITLE
Impossible to open a file when linear="no"

### ISSUE THREAD

**Issue by civodulab** [readium/kotlin-toolkit#124]
Impossible to open a file whose `itemref` attribute is `linear="no"`.
However it works in Thorium.

**Comment by mickael-menu** [comment]
Yes non linear resources are not yet supported, mostly because the correct behavior is unclear. See this discussion https://github.com/readium/architecture/discussions/143

What do you personally expect should happen?
* Opening a pop-up with the non linear resource inside.
* Replacing the currently visible resource by the non linear one, then how do you go back?

Do you have a sample book with `linear="no"`?

**Comment by civodulab** [comment]
I am working on a book for visually impaired youngsters. There is a rather long notice explaining how the options work. I wanted to hide it to avoid too many pages to turn. This notice will only be read once or twice. On my configuration page I have a link to this manual and on the page of this manual I have a link to the configuration page.

Since I saw that the option existed I wanted to use it.

> What do you personally think should happen?
> Open a pop-up with the non-linear resource inside.
> Replace the currently visible resource with the non-linear resource, then how do you go back?

Replace the currently visible resource with the non-linear resource.

I made a very quick epub with a non-linear resource.
[test_linear_no.zip](https://github.com/readium/kotlin-toolkit/files/8752540/test_linear_no.zip)
.

**Comment by mickael-menu** [comment]
Thank you, such samples are very useful.

This feature is [a real can of worms](https://github.com/readium/architecture/discussions/143) so I doubt we will get a good implementation soon. But maybe we can work out a limited solution in the meantimes, as it's completely broken right now.

**Comment by qnga** [comment]
Closing as we've implemented a way for apps to handle non-linear resources and it's not clear if we'll do more.
